lib/views/recent-commits-view.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {autobind} from 'core-decorators';
+
+class RecentCommitView extends React.Component {
+  static propTypes = {
+    commit: PropTypes.object.isRequired,
+  };
+
+  render() {
+    return (
+      <li className='github-RecentCommit'>
+        <img className='github-RecentCommit-avatar'
+          src="https://avatars3.githubusercontent.com/u/7910250?v=4&s=32"
+          title={`@kuychaco - ${this.props.commit.getAuthorEmail()}`}
+        />
+        <span className='github-RecentCommit-message'>{this.props.commit.getMessage()}</span>
+        <time className='github-RecentCommit-time'
+          title={this.props.commit.getAuthorDate()}>
+          1h
```

Comment:
For the "time ago" format, to keep it narrow as possible, how about this:

```
<1m, 1m, 1h, 1d, 1w, 1M, 1y
```

So they are all lower case, except for the **month** to differentiate from **minutes**. When hovering and waiting for a bit, the title could show the date like `Feb 20, 2018` and maybe without time.